simpledotorg/bp-passport
# 2 Step API Authentication Process

To test you'll need to:

1. Run on a real device.
2. Go here: https://www.qr-code-generator.com/ and create a QR code with this id that will authenticate: '86d89f24-fc11-4829-aa4e-5daee20a370a'
3. You should also test against a bad QR code
4. When you get the stage 2 - the verify number screen - 000000 will work as the OTP to complete the authentication.
